src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1234,7 +1234,7 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                 };
                                 var removedChangeVector = tss.DeleteTimestampRange(context, deletionRangeRequest, rcvdChangeVector, updateMetadata: false);
                                 if (removedChangeVector != null)
-                                    context.LastDatabaseChangeVector = ChangeVectorUtils.MergeVectors(removedChangeVector, rcvdChangeVector);
+                                    context.LastDatabaseChangeVector = ChangeVectorUtils.MergeVectors(removedChangeVector, context.LastDatabaseChangeVector);
```

Comment:
I wonder if we missing setting the cv in case of `canDeleteEntireSegment` in `DeleteTimestampRange` ? Before the change we were returning a cv from `DeleteTimestampRange()` in case we were removing entire segment
https://github.com/shiranshalom/ravendb/blob/3375000801a6da8f1964dfdfbb70fa29437551b0/src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs#L489

Reply:
done